test/contrib/automotive/bmw/enet.uts

```diff
@@ -1,8 +1,7 @@
 + ENET Contrib tests
 
 = Load Contrib Layer
-
-load_contrib("automotive.bmw.enet")
+from scapy.contrib.automotive.bmw.enet import *
```

Comment:
So this means we will not test `load_contrib()`?

Reply:
I made this change before I discovered that `load_contrib(whatever, globals_dict=globals())` has the same effect.. I will change it.